.travis.yml

```diff
@@ -20,41 +20,28 @@ env:
 matrix:
   fast_finish: true
   include:
-    - php: 5.6
-      env:
-        - DEPS=lowest
-    - php: 5.6
-      env:
-        - DEPS=latest
-    - php: 7
-      env:
-        - DEPS=lowest
-    - php: 7
-      env:
-        - DEPS=latest
-    - php: 7.1
+    - php: 7.3
       env:
         - DEPS=lowest
-    - php: 7.1
+    - php: 7.3
       env:
         - DEPS=latest
         - CS_CHECK=true
         - TEST_COVERAGE=true
-    - php: 7.2
+    - php: 7.4
       env:
         - DEPS=lowest
-    - php: 7.2
+    - php: 7.4
       env:
         - DEPS=latest
-    - php: 7.3
+    - php: nightly
       env:
         - DEPS=lowest
-    - php: 7.3
-      env:
-        - DEPS=latest
-    - php: 7.4
+        - COMPOSER_ARGS="--no-interaction --ignore-platform-reqs"
+    - php: nightly
```

Comment:
At the moment version `8.0.1-dev` is used on Travis therefore it should be replaced with `8.0`.

See: https://travis-ci.com/github/laminas/laminas-form/jobs/457287514#L211

Reply:
Done

Reply:
@Slamdunk 
Please check Slack for new message. Thanks! 😃